devstack/exercises/nfp_service/user-data/visibility_user_data

```diff
@@ -76,3 +76,4 @@ write_files:
 
 runcmd:
   - cd /usr/share/visibility/docker/ && bash rundocker.sh
+  - docker run -d --restart=always --name configurator -it -p 5672:5672 -p 8070:8080 configurator-docker
```

Comment:
For preventing access from visibility external network  please add   --net=internalbr to the runcmd, 
 docker run -d  --restart=always  --name configurator -it --net=internalbr -p 5672:5672 -p 8070:8080 configurator-docker